src/pages/index.js

```diff
@@ -73,14 +110,38 @@ const createCard = (data) => {
   ).generateCard();
   return cardElement;
 };
+//
+userInfo.fetchAndSetUserInfo();
 
+api.getInitialCards().then((initialCards) => {
+  gallery = new Section(
+    {
+      data: initialCards,
+      renderer: (item) => {
+        const cardElement = createCard(item);
+        gallery.addItem(`append`, cardElement);
+      },
+    },
+    ".gallery__list"
+  );
+  gallery.renderItems();
+});
 // Edit profile modal
 editProfileButton.addEventListener("click", () => {
-  const userInfoData = userInfo.getUserInfo();
-  nameInput.value = userInfoData.name;
-  jobInput.value = userInfoData.job;
-  editPopupForm.open();
-  formValidators[editPopupForm.form.getAttribute("name")].resetValidation();
+  api.getUserInfo().then((data) => {
+    nameInput.value = data.name;
+    jobInput.value = data.about;
+    editPopupForm.open();
+    formValidators[editPopupForm.form.getAttribute("name")].resetValidation();
```

Comment:
`api.getUserInfo()` is not needed when you open the `profile` modal because you have ` const userInfoData = userInfo.getUserInfo();` for that (like it was in project 8)

It's a waste of resources